guides/get_started_faster_with_forge.textile

```diff
@@ -71,116 +48,113 @@ h3. Setup Arquillian
 
-A new test class with a deployment and test method is added to the project.
+div(filename). src/test/java/org/arquillian/tutorial/beans/MySimpleBeanTest.java
 
-div(filename). src/test/java/org/arquillian/example/MySimpleBeanTest.java
+bc(prettify).. package org.arquillian.tutorial.beans;
 
-bc(prettify).. package org.arquillian.example;
+               import org.arquillian.tutorial.beans.MySimpleBean;
 
-import org.arquillian.example.MySimpleBean;
-import javax.inject.Inject;
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.jboss.shrinkwrap.api.asset.EmptyAsset;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import static org.junit.Assert.*;
-import static org.hamcrest.core.Is.*;
+               import javax.inject.Inject;
 
-@RunWith(Arquillian.class)
-public class MySimpleBeanTest {
-    @Inject private MySimpleBean mysimplebean;
+               import org.jboss.arquillian.container.test.api.Deployment;
+               import org.jboss.arquillian.junit.Arquillian;
+               import org.jboss.shrinkwrap.api.ShrinkWrap;
+               import org.jboss.shrinkwrap.api.spec.JavaArchive;
+               import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+               import org.junit.Assert;
+               import org.junit.Test;
+               import org.junit.runner.RunWith;
 
-    @Deployment public static JavaArchive createDeployment() {
-        return ShrinkWrap.create(JavaArchive.class, "test.jar")
-            .addClass(MySimpleBean.class)
-            .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
-    }
+               import static org.junit.Assert.*;
+               import static org.hamcrest.core.Is.*;
 
-    @Test
-    public void testIsDeployed() {
-        Assert.assertNotNull(mysimplebean);
-    }
-}
+               @RunWith(Arquillian.class)
+               public class MySimpleBeanTest {
+
+               	@Inject
+               	private MySimpleBean mySimpleBean;
+
+               	@Deployment
+               	public static JavaArchive createDeployment() {
+               		return ShrinkWrap.create(JavaArchive.class)
+               				.addClass(MySimpleBean.class)
+               				.addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
+               	}
+
+               	@Test
+               	public void should_be_deployed() {
+               		Assert.assertNotNull(mySimpleBean);
+               	}
+               }
 
 p. As you can see, the test injects the class under test (i.e., @MySimpleBean@) using CDI, so this will immediately prove if the test really runs in a container.
 
-You can run the test from your IDE, from Maven (outside of Forge) or from Forge itself. Let's stick with Forge.
+You can run the test from your IDE, from Maven (outside of Forge) or from Forge itself. Let's run with mvn
```

Comment:
But if you say sometimes it works and sometimes it doesn't then it's a bit questionable if we should mention it at all.